packages/io/amf-serializer/README.md

```diff
@@ -1,15 +1,17 @@
 ## @jscad/amf-serializer
 
-> amf serializer for the jscad project (from CSG)
+> Serializer of JSCAD geometries to AMF shapes
```

Comment:
shapes can be omited, as amf is much more than shapes :)
 perhaps just adding a link to AMF definition instead (https://en.wikipedia.org/wiki/Additive_manufacturing_file_format)

Reply:
Done